roles/generate-jenkins/templates/Jenkinsfile.j2

```diff
@@ -1368,16 +1367,29 @@ pipeline {
               echo "$escaped_table"
             }
 
-            # Retrieve JSON data from URL
-            data=$(get_json "$CI_JSON_URL")
-            # Create table from JSON data
-            table=$(build_table "$data")
-            echo -e "$table"
+            if [[ "${CI}" = "true" ]]; then
+              # Retrieve JSON data from URL
+              data=$(get_json "$CI_JSON_URL")
+              # Create table from JSON data
+              table=$(build_table "$data")
+              echo -e "$table"
 
-            curl -X POST -H "Authorization: token $GITHUB_TOKEN" \
-              -H "Accept: application/vnd.github.v3+json" \
-              "https://api.github.com/repos/$LS_USER/$LS_REPO/issues/$PULL_REQUEST/comments" \
-              -d "{\\"body\\": \\"I am a bot, here are the test results for this PR: \\n${CI_URL}\\n${SHELLCHECK_URL}\\n${table}\\"}"'''
+              curl -X POST -H "Authorization: token $GITHUB_TOKEN" \
+                -H "Accept: application/vnd.github.v3+json" \
+                "https://api.github.com/repos/$LS_USER/$LS_REPO/issues/$PULL_REQUEST/comments" \
+                -d "{\\"body\\": \\"I am a bot, here are the test results for this PR: \\n${CI_URL}\\n${SHELLCHECK_URL}\\n${table}\\"}"
+            else
+              if [[ "${MULTIARCH}" = "true" ]]; then
+                PR_COMMENT_IMAGES="\\`${GITHUBIMAGE}:amd64-${META_TAG}\\`\\n{% if build_armhf %}\\`${GITHUBIMAGE}:arm32v7-${META_TAG}\\`\\n{% endif %}\\`${GITHUBIMAGE}:arm64v8-${META_TAG}\\`"
+              else
+                PR_COMMENT_IMAGES="\\`${GITHUBIMAGE}:${META_TAG}\\`"
+              fi
```

Comment:
I would just do the multiarch tag in either case, don't think there's any need to do in the individual arches as 9/10 we'll pull the multiarch manifest and in cases where we do need to pull a specific arch we know how.

Also means you don't have to special-case armhf and then remove it in a few months.